diagnostics/app/controllers/DiagnosticsController.scala

```diff
@@ -43,7 +43,7 @@ object DiagnosticsController extends Controller with Logging {
   }
 
   def csp = Action(jsonParser) { implicit request =>
-    if (conf.switches.Switches.CspReporting.isSwitchedOn && r.nextInt(100) == 1) {
```

Comment:
Let's check how many reports are ingested into ES when we turn this on. 
I don't think the volume of reports would be big deal but it worths checking